### ISSUE TITLE
Error start plugman 2.1.2 in Spigot 1.11.2

### ISSUE THREAD

**Issue by C4BR3R4** [r-clancy/PlugMan#41]
Error starting plugin in Spigot 1.11.2
https://hastebin.com/newisusoca.sql

**Comment by smhanes** [comment]
Same as C4BR3R4. Restarted server this morning for reg. maint. and before I did noticed PlugMan had mentioned an update. Upon reboot, I noticed the same errors. The server does start and loads all other plugins.

**Comment by r-clancy** [comment]
I'm not sure what happened to the last version, the copy of the JAR I was testing with locally didn't throw any of those exceptions - I also validated that the plugin.yml was valid as well.

I was able to re-produce using the uploaded version, so I have pulled version 2.1.2 from dev.bukkit.org and have uploaded 2.1.3 (a re-build) that seems to have the issue fixed - hopefully this will be approved later today.

Thanks for reporting this so quickly!
